src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -62,76 +91,111 @@ class UpdateMarkupModelBase extends React.Component<Props, State> {
                 source_uuids: current.sources.map(provider => provider.uuid),
                 source_type: current.source_type === 'OpenShift Container Platform' ? 'OCP' : 'AWS',
                 markup: {
-                  value: this.state.markup,
+                  value: this.state.sign === '-' ? this.state.sign + this.state.markup : this.state.markup,
                   unit: 'percent',
                 },
               };
               updateCostModel(current.uuid, newState, 'updateMarkup');
             }}
             isDisabled={
               isNaN(Number(this.state.markup)) ||
-              Number(this.state.markup) === Number(current.markup.value || 0) ||
+              (Number(this.state.markup) ===
+                Number(
+                  current.markup.value.startsWith('-') ? current.markup.value.substring(1) : current.markup.value || 0
+                ) &&
+                this.state.sign === this.state.origSign) ||
               isLoading
             }
           >
-            {t('cost_models_details.add_rate_modal.save')}
+            {t('save')}
```

Comment:
Mocks say "Save changes"

Reply:
Yes It does and I talked to Natalie about that, we decided Save was fine